device/iot-device-client/pom.xml

```diff
@@ -65,7 +65,7 @@
         <dependency>
             <groupId>org.eclipse.paho</groupId>
             <artifactId>org.eclipse.paho.client.mqttv3</artifactId>
-            <version>1.2.2</version>
```

Comment:
Question - doesn't device-client take a dependency on deps? Why do we have this added in both?

Reply:
Good question! We don't need it declared in device client's pom, so I'll get rid of it.